# Gene-expression-data-analysis

Potential interactions between estrogen receptor (ER) and RAD21 in MCF7 breast cancer cells were examined. Breast cancer is the most commonly diagnosed cancer in the female population and 70% of breast cancers express ER. ER agonism contributes to the cell cycle progression of ER-positive breast cancer cells by regulating transcription of target genes. RAD21 is a subunit of cohesin and is involved in holding sister chromatids together, DNA repair, and gene regulation. RAD21 controls gene expression by bringing distal DNA elements into proximity. MCF7 is a breast cancer cell line that has been broadly used for cancer research, and genome wide studies in MCF7 cells have shown that cohesin co-binds to DNA with ER at numerous sites. Determining which genes are differentially expressed between estrogen treatment vs. control and estrogen treatment with RAD21 knockdown vs. control indicates potential genes that are estrogen responsive whose expression is influenced by RAD21. 

<!-- wp:image {"id":9211,"sizeSlug":"large"} -->
<figure class="wp-block-image size-large"><img src="https://gtbinf.files.wordpress.com/2019/11/workflow2.png?w=784" alt="" class="wp-image-9211"/><figcaption> Figure 1. Workflow of gene expression analysis of GSE114260 (RNA-Seq) and GSE59908 (microarray) datasets. </figcaption></figure>
<!-- /wp:image -->

Datasets GSE114260 and GSE59908 were analyzed. Dataset GSE114260, “Effect of estrogen (E2) treatment on the C4-12 relative to the MCF7 cells,” is composed of RNA-seq data from an Illumina HiSeq 2000 using cDNA made from polyA RNA. We assessed differential gene expression between vehicle (n =3; samples GSM3138572, GSM3138573, GSM3138574) and estrogen treated (48 hr; n=3; samples GSM3138575, GSM3138576, GSM3138577) MCF7 breast cancer cells. First, the SRA sample files were retreived from the Gene Expresison Omnius (GEO) and converted to fastq files and only reads with a quality score greater than 33 were used. Transcript abundance was quantified with Kallisto (2) using a reference transcriptome obtained from NCBI. In Sleuth, a likelihood ratio test and Wald test were performed and only genes that had a q ≤ 0.05 for both tests were kept. False discovery rate (FDR) values were calculated using the method of Benjamini and Hochberg. Prior to p-value calculation, genes were filtered to only include transcripts with a minimum of  five transcripts in at least 47% of samples. Differentially expressed genes were identified if fold change was ≥ 2, and q ≤ 0.05.

The list of differentially expressed genes was compared to data published by Petrossian et al. (2018). Petrossian et al. found a total of 2,595 differentially expressed genes. We found 2,857 using the same q value and fold change cutoffs. The full list of differentially expressed genes was not published. However, a list of 29 differentially expressed genes was published, and the top ten genes in terms of fold change were all found within our list (genes: B3GNT6, C5AR2, GREB1, GRIK3, HR, HSPB8, IGFBP4, PGLYRP2, PGR, and TUBA3E). Our list of differentially expressed genes represents genes which are likely estrogen responsive. Hierarchical clustering, heat mapping, and principal component analysis (PCA) were performed using ExAtlas (8). Samples from the same treatment all clustered together in the heatmap indicating that the treatment had an effect on gene expression and the PCA shows that 38% of variation is explained by principal component one and 57% is explained by components one and two. 

https://gtbinf.files.wordpress.com/2019/11/rna-seq-heatmap.png?w=1024

